eng/Versions.props

```diff
@@ -2,8 +2,8 @@
 <!-- Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the MIT license. See License.txt in the project root for full license information. -->
 <Project>
   <PropertyGroup>
-    <VersionPrefix>17.11.0</VersionPrefix>
-    <PackageValidationBaselineVersion>17.8.3</PackageValidationBaselineVersion>
+    <VersionPrefix>17.11.0</VersionPrefix><DotNetFinalVersionKind>release</DotNetFinalVersionKind>
```

Comment:
Could you please add this to the new line? 
Also in order to understand: What exactly it does?

Reply:
This is specifically a place where we DO NOT put a new line - for getting a merge conflict when we get it flowing to main branch. 
After adding this flag, we always need to bump a version if we need to merge anything, and the version considered stable.